routes/offers.js

```diff
@@ -1,75 +1,102 @@
-const express = require("express");
-const jwt = require("jsonwebtoken");
-const bodyParser = require("body-parser");
-require("dotenv").config();
+const Offer = require("../model/Offer");
+const router = require("express").Router();
+const verify = require("./verifyToken");
 
-const { getOffers, getOffer, createOffer } = require("../lib/offers");
+// Get all offers
+router.get("/", verify, async (req, res) => {
+  try {
+    const offers = await Offer.find();
+    if (!offers) return res.status(400).send("No offers available");
+    res.send(offers);
+  } catch (error) {
+    res.status(500).send("Internal server error");
+  }
```

Comment:
If something unexpected happens, you will not see the error message right now. Happens multiple times.

```suggestion
  } catch (error) {
    console.error(error);
    res.status(500).send("Internal server error");
  }
```

Reply:
Will fix this soon. Thank you so much!